regex/challenges-04.test.js

```diff
@@ -0,0 +1,230 @@
+'use strict';
+
+// ------------------------------------------------------------------------------------------------
+// CHALLENGE 1
+//
+// Write a function named isNum that takes in a string or number of any length.
+// This function should use a regular expression pattern to return true if the input
+// contains a number, and false if the input does not contain a number.
+//
+// Example input/output:
+// 12345 => true
+// '12345' => true
+// 'h3llo world' => true
+// 'hello world' => false
+// ------------------------------------------------------------------------------------------------
+
+const isNum = (num) => {
+  // Solution code here...
+  const re = /\d/;
+  if(re.test(num)) return true;
+  else return false;
+

```

Comment:
you can combine lines 20 and 21 to simply say `return re.test(num)` since the test() method returns a boolean